### PR TITLE
ci: enable codecov

### PR DESCRIPTION
**Pull request by jibel**

### CODE REVIEW

**Review by didrocks — APPROVED** [review]
This is the correct file to upload ;)

File: .github/workflows/qa.yaml [hunk 1]
```diff
@@ -231,7 +231,7 @@ jobs:
...
         run: |
           go test -shuffle=on ./... -race -tags=gowslmock
-      #- name: Upload coverage to Codecov
-      #  uses: codecov/codecov-action@v3
-      #  with:
-      #    file: /tmp/coverage.out
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          file: /tmp/coverage.out
```
Comment (CarlosNihelton) [inline .github/workflows/qa.yaml:237]: The input option is `files` :p

https://github.com/codecov/codecov-action?tab=readme-ov-file#arguments

**Review by CarlosNihelton — CHANGES_REQUESTED** [review]
I think I found a small mistake.

File: .github/workflows/qa.yaml (hunk 1 above)
Comment (didrocks) [reply .github/workflows/qa.yaml:237]: Interesting, we are still using the `file:` in adsys which definitively works (https://github.com/ubuntu/adsys/blob/main/.github/workflows/qa.yaml#L80), but probably an initial option that has been deprecated and then unlisted. Let’s use the new format then!

**Review by EduardGomezEscandell — APPROVED** [review]
👍 

Looks like the gRPC action is rate-limiting us again 🙄. This PR does not touch any code so we can ignore the failure.

**Pull request merged**